### DISCUSSION TITLE
When will a new version of release come out?

### DISCUSSION THREAD

**Discussion by TensorP7** [iina/iina#5091] [General]
I recently saw our hardworking developers added a bunch of updates, some of them matters for a better experience like spatial audio support. So I was wondering when will the new version with these great features come out. Will it be out in September along with new macOS Sequoia?

**Comment by low-batt** [comment]
I'm a junior IINA developer and not always in the know as to when a release will be cut. And any future release plans are always subject to change…

IINA is currently under heavy development for a long planned feature release. Before this is officially released there will be beta releases. Along with new features and major updates to libraries IINA is dependent upon such as mpv and FFmpeg, this release will include fixes that were deemed too risky to put into a release that does not include beta releases.

There might be a minor release of IINA before the first feature release beta to address 1.3.5 regressions, but the focus of the team is on the feature release.

At this time there is still open work items on the schedule for beta one. I'm about to add another issue to the schedule after discovering a new regression. We still are not in the phase of a release where exactly what needs to be done for the release has been nailed down.

The nightly build contains support for Spatial Audio as IINA has been changed to use the new mpv AVFoundation audio driver. However we have encountered some bad behavior with that driver, such as mute having a 2 second delay. That means we are going to have to add UI for a setting that allows the user to choose the audio driver and continue to use the current Core Audio driver by default. The AVFoundation driver will be listed as "experimental". I will be doing that soon. Hopefully mpv will address the AVFoundation glitches for a future IINA release.

With respects to macOS Sequoia, IINA and other players are broken in Sequoia beta 4. Hopefully Apple will address this regression in the next beta. However when this happened in macOS Ventura it took Apple a couple of betas before the regression was fixed. Normally IINA releases are not tied to a macOS release schedule. If Apple makes a change that is not backward compatible requiring applications to change then it becomes a consideration for the IINA schedule.

Given the risky fixes to critical IINA code that are already in the nightly build the development team needs to not rush this next release of IINA. That said, the team has already started postponing adding some features until beta 2 and postponing some features until the next feature release in order to get beta 1 out the door sooner.

I've not checked in with the other developers to see where they are with the tasks they are working on for beta 1. I'm getting closer to finishing what I plan on doing for beta 1. When I finish my remaining tasks I will turn by attention to testing which could generate more work if I find regressions.

**Reply by TensorP7** [reply]
Thanks for all the incredible work you’ve put into developing IINA. I am looking forward to the upcoming IINA release. Now I understand the challenges faced with macOS Sequoia and hope that Apple addresses these compatibility issues soon. I appreciate that despite these challenges, you are not rushing the release but instead focusing on ensuring quality and stability. Thank you again for your work.
